web-ui/src/main/resources/catalog/templates/editor/directory.html

```diff
@@ -117,48 +314,78 @@
               <div data-groups-combo="" data-owner-group="importData.group" lang="lang"
                    data-groups="groups" data-exclude-special-groups="false"/>
             </div>
-
-            <div class="btn-toolbar">
-              <button type="button" class="btn btn-primary pull-right"
-                      data-ng-click="importEntry(xml)"
-                      data-ng-disabled="xml === ''"
-                      title="{{'importEntry' | translate}}">
-                <i class="fa fa-plus"/>&nbsp;
-              </button>
-            </div>
           </form>
         </div>
       </div>
     </div>
-    <div class="col-sm-8" data-ng-if="activeEntry">
-      <div class="panel-default panel">
+  </div>
+
+  <!-- full editor is only used for edition -->
+  <div class="row" ng-show="currentEditorAction != '' && activeEntry">
+    <div class="col-sm-12 col-md-8">
+      <div class="panel-default panel entry-editor">
         <div class="panel-heading">
-          <span data-translate="">{{activeEntry.title || activeEntry.defaultTitle}}</span>
-          <div class="pull-right">
-            <div data-gn-metadata-group-updater="activeEntry.groupOwner"
-                 data-metadata-id="activeEntry['geonet:info'].id"/>
+
+          <span translate class="entry-editor-title">
+            directoryEntryEditor</span>
+
+          <div class="flex-spacer"></div>
+
+          <div class="entry-editor-controls">
+            <div gn-metadata-group-updater="activeEntry.groupOwner"
+                 metadata-id="activeEntry['geonet:info'].id">
+            </div>
+
+            <div class="btn-group">
+              <button type="button" class="btn btn-default dropdown-toggle"
+                  data-toggle="dropdown"
+                  ng-disabled="activeEntry['geonet:info'].edit !== 'true'"
+                  ng-class="{'btn-primary': activeEntry.valid==1, 'btn-danger': !activeEntry.valid}">
+                <i class="fa fa-check"/>
+                <span translate class="hidden-xs hidden-sm hidden-md">
+                  isValid</span>
+                &nbsp;
+                <i class="caret"/>
+              </button>
+              <ul class="dropdown-menu pull-right" role="menu">
```

Comment:
Valid and reject should be the same menu --- at least disable valid if already valid, and same for invalid ?